<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbletonDialog</class>
 <widget class="QDialog" name="AbletonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instruments</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="connectionBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Not Connected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connectButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="synchronizeButton">
     <property name="text">
      <string>Synchronize</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>AbletonDialog</receiver>
   <slot>onConnectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>synchronizeButton</sender>
   <signal>clicked()</signal>
   <receiver>AbletonDialog</receiver>
   <slot>onSynchronizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>AbletonDialog</receiver>
   <slot>onButtonBoxClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectClicked()</slot>
  <slot>onSynchronizeClicked()</slot>
  <slot>onButtonBoxClicked(QAbstractButton*)</slot>
 </slots>
</ui>
